velox/functions/lib/Re2Functions.cpp

```diff
@@ -239,7 +239,7 @@ class Re2Match final : public VectorFunction {
   }
 };
 
-void checkForBadGroupId(int groupId, const RE2& re) {
+void checkForBadGroupId(int64_t groupId, const RE2& re) {
```

Comment:
Does this have to be int64 because presto supports bigints for groups ?

Reply:
select regexp_extract_all('1a 2b 14m', '(\d+)([a-z]+)', cast (2 as  bigint));